packages/auth/src/providers/cognito/utils/oauth/getRedirectUrl.native.ts

```diff
@@ -10,7 +10,12 @@ export function getRedirectUrl(redirects: string[]): string {
 			!redirect.startsWith('http://') && !redirect.startsWith('https://')
 	);
 	if (!redirect) {
-		throw invalidRedirectException;
-	}
+		if (redirects.length === 0) {
+			throw invalidRedirectException;
+		} else {
+			// fallsback to available URL https://github.com/aws-amplify/amplify-js/issues/12890
```

Comment:
```suggestion
			// fallsback to available redirect URL. Addresses: https://github.com/aws-amplify/amplify-js/issues/12890
```
or something on those lines 😅